build.cake

```diff
@@ -133,7 +133,31 @@ Task("PublishCalamariProjects")
         }
 });
 
+Task("PublishSashimiTestProjects")
```

Comment:
The Calamari packing above puts all runtimes into the same ZIP file, making all Teamcity agents pull down other runtimes as well. We may optimize this by creating separate ZIP's for different runtimes